client/src/components/SignUpPage.vue

```diff
@@ -4,16 +4,17 @@
     <div class="container">
       <div class="signup-form-container">
         <form>
-          <input class="input-textarea" type="text" placeholder="E-mail" />
-          <input class="input-textarea" type="text" placeholder="Passsword" />
+          <input class="input-textarea" type="text" placeholder="E-mail" v-model="email"/>
+          <input class="input-textarea" type="text" placeholder="Passsword" v-model="password"/>
           <input
             v-if="isCurrentPathSignUp"
             class="input-textarea"
             type="text"
```

Comment:
we NEVER use plaintext input for passwords

Reply:
Issue #24